php/index.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use Ionic\CustomAuthentication;
+
+require 'vendor/autoload.php';
+
+Flight::route('/auth',function () {
+    try {
+        $redirect_uri = CustomAuthentication::process();
+        Flight::redirect($redirect_uri);
+    } catch (\Exception $e) {
+        Flight::json(['error' => $e->getMessage(), 'code' => $e->getCode()], 401);
+    }
+});
+
+Flight::start();
+
+?>
```

Comment:
You are [correct](http://php.net/manual/en/language.basic-syntax.phptags.php). "If a file is pure PHP code, it is preferable to omit the PHP closing tag at the end of the file."